src/main/resources/static/_shared.yml

```diff
@@ -4331,7 +4330,7 @@ components:
         - decisionDetail
         - createdAt
         - applicationId
-        - createdByUserId
+        - createdByUser
```

Comment:
I wonder if we should be more explicit here to make it clear that appeal was `arbitratedBy` the given user? That the person reporting the appeal decision is always and necessarily the person who ruled on the appeal.

If we use `createdBy` there is perhaps sufficient ambiguity that it may look like the this appeal decision is being reported by someone else? That can't be the case in reality.

Reply:
Yeah, I did consider this, but thought it probably wasn't worth unpicking all the database columns etc. Happy to do this in a different PR if you think it's worth doing